UE4SS/src/LuaType/LuaUEnum.cpp

```diff
@@ -110,16 +110,199 @@ No overload found for function 'UEnum.ForEachName'.
+            // Param #2: Enum key index
+            std::string error_overload_not_found{R"(
+No overload found for function 'UEnum.InsertIntoNames'.
+Overloads:
+#1: InsertIntoNames(string Name, integer Index) // shiftValues = true
+#1: InsertIntoNames(string Name, integer Index, bool shiftValues)"};
+
+            int32_t stack_size = lua.get_stack_size();
+
+            if (stack_size <= 0)
+            {
+                lua.throw_error("Function 'UEnum.InsertIntoNames' cannot be called with 0 parameters.");
+            }
+
+            StringType param_name = L"";
+            int param_index = 0;
+            bool param_shift = true;
+
+            // P1 (Name), string
+            if (lua.is_string())
+            {
+                param_name = to_wstring(lua.get_string());
+            }
+            else
+            {
+                lua.throw_error("ERROR: Could not load parameter for \"Name\"");
+            }
+
+            // P2 (Index), integer
+            if (lua.is_integer())
+            {
+                param_index = lua.get_integer();
+            }
+            else
+            {
+                lua.throw_error("ERROR: Could not load parameter for \"Index\"");
+            }
+
+            // (Optional) P3 (ShiftValues), boolean
+            if (lua.is_bool())
+            {
+                param_index = lua.get_bool();
+            }
+
+            auto& lua_object = lua.get_userdata<UEnum>();
+
+            Unreal::FName key = Unreal::FName(param_name, Unreal::FNAME_Add);
+            auto pair = Unreal::TPair{key, Unreal::int64(param_index)};
+
+            lua_object.get_remote_cpp_object()->InsertIntoNames(pair, param_index, true);
+            return 1;
+        });
+
+        table.add_pair("EditNameAt", [](const LuaMadeSimple::Lua& lua) -> int {
+            // Function: EditNameAt
+            // Param #1: Enum key index
+            // Param #2: New name for index
+            std::string error_overload_not_found{R"(
+No overload found for function 'UEnum.EditNameAt'.
+Overloads:
+#1: EditNameAt(integer Index, string NewName))"};
+
+            int32_t stack_size = lua.get_stack_size();
+
+            if (stack_size <= 0)
+            {
+                lua.throw_error("Function 'UEnum.EditValueAt' cannot be called with 0 parameters.");
+            }
+
+            int param_index = 0;
+            std::string param_new_name = L"";
+
+            // P1 (Index), integer
+            if (lua.is_integer())
+            {
+                param_index = lua.get_integer();
+            }
+            else
+            {
+                lua.throw_error("ERROR: Could not load parameter for \"Index\"");
+            }
+
+            // P2 (NewName), string
+            if (lua.is_integer())
+            {
+                param_new_name = to_wstring(lua.get_string());
+            }
+            else
+            {
+                lua.throw_error("ERROR: Could not load parameter for \"NewName\"");
```

Comment:
This error message is not descriptive enough.
Add which function is throwing the error.